### ISSUE TITLE
An error was thrown during Nuxi initialization.

### ISSUE THREAD

**Issue by mercs600** [nuxt/nuxt#19125]
### Environment

------------------------------
- Operating System: `Darwin`
- Node Version:     `v18.14.1`
- Nuxt Version:     `0.0.0`
- Nitro Version:    `-`
- Package Manager:  `unknown`
- Builder:          `webpack`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------

### Reproduction

just type 
`npx nuxi init test-app`

### Describe the bug

After upgrading to 3.2.1, I encountered bugs while creating the project with Nuxi.
3.2.0 works correct 

### Logs

```shell-script
➜  Poligon npx nuxi init test-app
Nuxi 3.2.1                                                                                                                 17:15:58

 ERROR  TypeError: Cannot convert undefined or null to object                                                              17:15:59
```

**Comment by danielroe** [comment]
Thanks for the super-quick spot! It seems there was a last-minute upstream regression in `giget`. PR to fix: https://github.com/unjs/giget/pull/83

**Comment by denishacquin** [comment]
Same issue here, can't create a project.

**Comment by danielroe** [comment]
This should be resolved in v3.2.2 ❤️

**Comment by danielroe** [comment]
If you are experiencing a similar issue, the best thing is to open a new issue with a minimal reproduction.
